packages/webdriverio/src/commands/element/click.ts

```diff
@@ -133,7 +159,7 @@ async function actionClick(element: WebdriverIO.Element, options: Partial<ClickO
         skipRelease: false,
     }
 
-    const { button, x, y, skipRelease }: ClickOptions = { ...defaultOptions, ...options }
+    const { button = 0, x, y, skipRelease, duration = 0 }: ClickOptions = { ...defaultOptions, ...options }
```

Comment:
iirc I built-in this default behaviour in the actions API. We can do it here too but I don't believe we have to. This is not a blocker for the PR and can be resolved if you would like to keep it as is 👍

Reply:
I missed the partial so reverted my change and added it to the defaults. Thanks for mentioning it